modules/oneVideoBidAdapter.js

```diff
@@ -115,7 +116,7 @@ export const spec = {
       },
       {
         type: 'image',
-        url: spec.SYNC_ENDPOINT3
+        url: `https://sync-tm.everesttech.net/upi/pid/m7y5t93k?gdpr_qparam==${gdprApplies ? 1 : 0}&gdpr_consent_qparam=${consentString}&redir=https%3A%2F%2Fpixel.advertising.com%2Fups%2F55986%2Fsync%3Fuid%3D%24%7BUSER_ID%7D%26_origin%3D0%26gdpr%3D{gdpr}%26gdpr_consent%3D{gdpr_consent}`
```

Comment:
I don't think this is what you want:
1) gdpr_qparam as two equal signs
2) The macros at the end gdpr%3D{gdpr}%26gdpr_consent%3D{gdpr_consent} are not going to get resolved. If you're using gdpr_qparam instead of gdpr, then you can just drop the ones at the end.

Reply:
regarding the macros will check with the publisher once and will do the change this is the pixel which was given by 'everesttech' team.